dictionaries/software-terms/cspell.config.yaml

```diff
@@ -11,3 +11,10 @@ ignoreWords:
 ignorePaths:
   - CHANGELOG.md
   - dict
+patterns:
+  - name: term-comment
+    pattern: '/#.*/g'
```

Comment:
I love the fact you had to tell cspell to avoid checking its source content.

It's somehow cSpellception

Reply:
Yes, not all of the comments were proper English.